### ISSUE TITLE
Package should not be annotated @CheckReturnValue -- generated protobuf code fails Error Prone checks

### ISSUE THREAD

**Issue by orischwartz** [google/safe-html-types#8]
Package `com.google.common.html.types` is annotated with `@CheckReturnValue`: https://github.com/google/safe-html-types/blob/abccc1eb91b4dd1ca817c61f3c743fb3a3320fb6/types/src/main/java/com/google/common/html/types/package-info.java#L23

This causes builds that enforce the annotation (like [Error Prone](http://errorprone.info/bugpattern/CheckReturnValue)) to fail on generated protocol buffers which reference the safe html types defined in this repository.

Example: 
```proto
import "webutil/html/types/html.proto";

message Test {
  optional webutil.html.types.SafeUrlProto url_field = 1;
}
```

Attempting to compile with error prone will fail on the generated java protobuf code with a message like:

```
error: [CheckReturnValue] Ignored return value of method that is annotated with @CheckReturnValue
                subBuilder.mergeFrom(urlField_);
                                    ^
    (see http://errorprone.info/bugpattern/CheckReturnValue)
  Did you mean 'subBuilder = subBuilder.mergeFrom(urlField_);'?
```
```
error: [CheckReturnValue] Ignored return value of method that is annotated with @CheckReturnValue
    com.google.common.html.types.Html.getDescriptor();
                                                   ^
    (see http://errorprone.info/bugpattern/CheckReturnValue)
  Did you mean to remove this line?
```

Because the protocol buffer messages reside in the same `java_package` `com.google.common.html.types`, I don't believe there's a quick fix for this other than removing the annotation from from the package and instead annotating all classes in `types/src/main/java/com/google/common/html/types` individually. 

Update: This actually also affects several of the non-protobuf classes--`@CheckReturnValue` shouldn't apply to the following builders:
* `SafeHtmlBuilder`
* `SafeStyleBuilder`
* `TrustedResourceUrlBuilder`

**Comment by orischwartz** [comment]
Fixed via 7e6e902e52b206d1c369cc834f98e93911613aee

**Comment by mikesamuel** [comment]
Pushed [1.0.8]( http://search.maven.org/#artifactdetails%7Ccom.google.common.html.types%7Cparent%7C1.0.8%7Cpom) to maven.  It's there but it'll take some time to show up in maven search results.
